CITATION.cff

```diff
@@ -0,0 +1,181 @@
+# This CITATION.cff file was generated with cffinit.
+# Visit https://bit.ly/cffinit to generate yours today!
+
+cff-version: 1.2.0
+title: MESSAGEix
+message: >-
+  Thank you for using the MESSAGEix framework! Please cite
+  it using the metadata from this file :)
```

Comment:
I like it but maybe leave out the smiley face ;)
```suggestion
  it using the metadata from this file
```

Reply:
Aw :(